packages/geolocator/tizen/src/geolocator_tizen_plugin.cc

```diff
@@ -186,7 +205,27 @@ class GeolocatorTizenPlugin : public flutter::Plugin {
 
   void OnCancelGeolocatorServiceUpdates() {
     geolocator_service_updates_event_sink_ = nullptr;
-    TizenResult tizen_result = location_manager_->UnsetOnServiceStateChanged();
+    location_manager_->UnsetOnServiceStateChanged();
+  }
+
+  void OnListenGeolocatorUpdates(
```

Comment:
Any special reason to make separately?  Can't we do it all at once?

Reply:
Well, I'd appreciate it if you could elaborate on your proposal.

Reply:
never mind. It's wrong position!

Reply:
My first intention is here: 
https://github.com/flutter-tizen/plugins/blob/171542aaed62f0e87df6ae43150b2844cbf2aad8/packages/geolocator/tizen/src/geolocator_tizen_plugin.cc#L74